landing-pages/site/content/en/announcements/_index.md

```diff
@@ -13,6 +13,12 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# June 12, 2023
+
+Airflow PMC welcomes new Airflow Committer:
+
+* **Pankaj Singh** ([@pankajastro](https://github.com/pankajastro)
+
```

Comment:
For the announcement, I am not sure if it should be just on the social media (Linkedin and twitter) or on the site too

Reply:
We always add it to the site

Reply:
That being said, I think we forgot to add yours @hussein-awala , could you create a PR

Reply:
Ah okay, I thought something had changed after creating the official page on Linkedin, I will create the PR.